### DISCUSSION TITLE
How to change from high performance power plan

### DISCUSSION THREAD

**Discussion by m10th** [LeDragoX/Win-Debloat-Tools#166] [Q&A] [answered]
I applied the tweaks, saw that the battery of my laptop drained faster and reverted the tweaks to see if that was the cause. Looking at the Energy and Power configuration, the option to change the power plan is not there, it only says that "can't optimise when in high performance mode".

**Comment by LeDragoX** [comment, marked as answer]
It don't change the power plan back, as it didn't knew what power plan was before. I'll assume you were using the balanced pp, as it's the default one.
Open cmd or powershell as admin and paste this command to change to Balanced.
```cmd
powercfg /setactive 381b4222-f694-41f0-9685-ff5bb260df2e
```
